website/content/docs/connect/config-entries/service-splitter.mdx

```diff
@@ -1,54 +1,569 @@
+      serviceSubset: v2
+      namespace: target-namespace
+      partition: target-partition
+      requestHeaders:
+        set:
+          x-web-version: from-v2
+      responseHeaders:
+        set:
+          x-web-version: to-v2      
+```
+
+</Tab>
+
+</Tabs>
+
+## Specification
+
+This section provides details about the fields you can configure in the service splitter configuration entry.
+
+<Tabs>
+
+<Tab heading="HCL" group="hcl">
+
+### `Kind` 
+
+Specifies the type of configuration entry to implement. 
+
+#### Values
+
+- Default: none
+- This field is required
+- Data type: String value that must be set to `service-splitter` in HCL or `serviceSplitter` in YAML. 
+
+### `Name`
+
+Specifies a name for the configuration entry. The name is metadata that you can use to reference the configuration entry when performing operations, such as reading or writing configuration entry configuration entry settings. 
+
+#### Values
+
+- Default: Name of the node 
+- This field is required
+- Data type: String
+
+
+### `Namespace` <EnterpriseAlert inline />
+
+Specifies the [namespace](/consul/docs/enterprise/namespaces) to apply the configuration entry.
+
+#### Values
+
+- Default: None
+- Data type: String
+
+### `Partition` <EnterpriseAlert inline />
+Specifies the [admin partition](consul/docs/enterprise/admin-partitions) to apply the configuration entry.
+
+#### Values
+
+- Default: `Default`   
+- Data type: String
+
+### `Meta`
+
+Specifies key-value pairs to add to the KV store.
+
+#### Values
+
+- Default: none 
+- Data type: Map of one or more key-value pairs 
+  - keys: String
+  - values: String, integer, or float 
+
+### `Splits`
+
+Defines how much traffic to send to sets of service instances during a traffic split. 
+
+#### Values 
+
+- Default: None
+- This field is required for defining service splits
+- Data type: list of objects that can contain the following fields:
+  - `Weight`: The sum of weights for a set of service instances must add up to 100. 
+  - `Service`: This field is required to direct split traffic   
+  - `ServiceSubset`
+  - `Namespace` <Enterprise inline/>
+  - `Partition` <Enterprise inline/>
+  - `RequestHeaders`
+  - `ResponseHeaders`  
+
+### `Splits[].Weight`
+
+Specifies the percentage of traffic sent to the set of service instances specified in the [`Service`](#service) field. Each weight must be a floating integer between `0` and `100`. The smallest representable value is `.01`. The sum of weights across all splits must add up to `100`.
+
+#### Values
+
+- Default: `null`
+- This field is required for defining service splits
+- Data type: Floating integer from `.01` to `100`
+
+### `Splits[].Service`
+
+Specifies the name of the service to resolve.
+
+#### Values
+
+- Default: The service matching the configuration entry [`Name`](#name) field.
+- Data type: String
+
+### `Splits[].ServiceSubset`
+
+Specifies a subset of the service to resolve. This field overrides the `DefaultSubset`. 
```

Comment:
```suggestion
Specifies a subset of the service to resolve. A service subset assigns a name to a specific subset of discoverable service instances within a datacenter, such as `version2` or `canary`. All services have an unnamed default subset that returns all healthy instances. 

You can define service subsets in a [service resolver configuration entry](/consul/docs/connect/config-entries/service-resolver), which are referenced by their names throughout the other configuration entries. This field overrides the default subset value in the service resolver configuration entry.   
```
Thanks for catching this - I was stumped for a while and had forgotten to come back to it.